Entity/SelectedStopPoint.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace CanalTP\MttBundle\Entity;
+
+/**
+ * Class SelectedStopPoint
+ */
+class SelectedStopPoint extends AbstractEntity
+{
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var string externalRouteId
+     */
+    private $externalRouteId;
+
+    /**
+     * @var string externalStopPointId
+     */
+    private $externalStopPointId;
+
+    /**
+     * @var integer rank
+     */
+    private $rank;
+
+    /**
+     * @var LineTimetable $lineTimetable
+     */
+    private $lineTimetable;
+
+    /**
+     * Get id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set externalRouteId
+     *
+     * @param string $externalRouteId
+     * @return SelectedStopPoint
+     */
+    public function setExternalRouteId($externalRouteId)
+    {
+        $this->externalRouteId = $externalRouteId;
+
+        return $this;
+    }
+
+    /**
+     * Get externalRouteId
+     *
+     * @return string
+     */
+    public function getExternalRouteId()
+    {
+        return $this->externalRouteId;
+    }
+
+    /**
+     * Set externalStopPointId
+     *
+     * @param string $externalStopPointId
+     * @return SelectedStopPoint
+     */
+    public function setExternalStopPointId($externalStopPointId)
+    {
+        $this->externalStopPointId = $externalStopPointId;
+
+        return $this;
+    }
+
+    /**
+     * Get externalStopPointId
+     *
+     * @return string
+     */
+    public function getExternalStopPointId()
+    {
+        return $this->externalStopPointId;
+    }
+
+    /**
+     * Set rank
+     *
+     * @param string $rank
```

Comment:
And if it comes from Navitia, maybe to be more consistent with the existing tables it should be named external_rank or even sthg more precise like external_rank_id no?